db/db/src/main/java/io/helidon/db/DbExecute.java

```diff
@@ -0,0 +1,407 @@
+        return createNamedDelete(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute delete statement using a statement passed as an argument.
+     *
+     * @param statement  the delete statement to be executed
+     * @param parameters query parameters to set
+     * @return number of rows deleted from the database
+     */
+    default CompletionStage<Long> delete(String statement, Object... parameters) {
+        return createDelete(statement).params(parameters).execute();
+    }
+
+    /*
+     * DML
+     */
+
+    /**
+     * Create a data modification statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return data modification statement
+     */
+    DbStatement<?, CompletionStage<Long>> createNamedDmlStatement(String statementName, String statement);
+
+    /**
+     * Create a data modification statement using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @return data modification statement
+     */
+    DbStatement<?, CompletionStage<Long>> createNamedDmlStatement(String statementName);
+
+    /**
+     * Create a data modification statement using a statement passed as an argument.
+     *
+     * @param statement the data modification statement to be executed
+     * @return data modification statement
+     */
+    DbStatement<?, CompletionStage<Long>> createDmlStatement(String statement);
+
+    /**
+     * Create and execute a data modification statement using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @param parameters    query parameters to set
+     * @return number of rows modified
+     */
+    default CompletionStage<Long> namedDml(String statementName, Object... parameters) {
+        return createNamedDelete(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute data modification statement using a statement passed as an argument.
+     *
+     * @param statement  the delete statement to be executed
+     * @param parameters query parameters to set
+     * @return number of rows modified
+     */
+    default CompletionStage<Long> dml(String statement, Object... parameters) {
+        return createDelete(statement).params(parameters).execute();
+    }
+
+    /*
+     * UNKNOWN
+     */
+
+    /**
+     * Create a generic database statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return generic database statement that can return any result
+     */
+    DbStatement<?, DbResult> createNamedStatement(String statementName, String statement);
```

Comment:
Another occurence of _hybrid_ method with both statement name and statement string.